charts/tenant-base/chart/Chart.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: tenant-base
+description: A basic Helm chart for RTP tenants
```

I don't think we should distinguish between SCCP/RTP tenants here.
```suggestion
description: A basic Helm chart for tenants
```